doc/dev/index.rst

```diff
@@ -227,16 +243,63 @@ Pending Backport Fix has been merged, backport(s) pending
 Resolved         Fix and backports (if any) have been merged
 ================ ===========================================
 
-If you do not have permission to change the status yourself, don't worry:
-someone will probably change it for you, even without asking. You can also
-ask for assistance on IRC.
+Submitting patches
+==================
+
+There are two ways to submit patches to Ceph: 
+
+1. `GitHub`_ pull requests
+2. Patch submission via :code:`ceph-devel@vger.kernel.org`
```

Comment:
For the sake of simplicity and because we don't use patch submission via mail (I don't know how to do that properly ;-), maybe we could just explain GitHub ?